test/unit/options.js

```diff
@@ -95,3 +96,35 @@ describe('generator options', function () {
     })
   })
 })
+
+describe('Run config Auditor', function () {
+  it('Should automatically run the config auditor', function () {
+    let ConfigAuditPassing = false
+
+    return helpers.run(path.join(__dirname, '../../generators/app'))
+      .withOptions({
+        'standard-install': true
+      })
+      .then(function () {
+        const testApp = exec(`node ${__dirname}/../../node_modules/roosevelt/lib/scripts/configAuditor.js`)
+
+        testApp.stdout.on('data', (data) => {
+          if (data.includes('Missing Dependency') && data.includes('rooseveltConfig audit completed with no errors found')) {
+            ConfigAuditPassing = true
+          }
+
+          if (data.includes('rooseveltConfig audit completed with no errors found')) {
+            ConfigAuditPassing = true
+          }
+        })
+
+        testApp.on('message', () => {
+          testApp.send('stop')
+        })
+
+        testApp.on('exit', () => {
+          assert.strictEqual(ConfigAuditPassing, true, 'rooseveltConfig audit completed with no errors found')
+        })
```

Comment:
This child process never actually gets killed anywhere in this test, because of that it's polluting the environment in other tests. The easiest way to fix this is to use [execSync](https://nodejs.org/api/child_process.html#child_process_child_process_execsync_command_options) instead, since this isn't a persistent process that we need to wait on events for.